src/main/java/uk/gov/hmcts/befta/featuretoggle/DefaultMultiSourceFeatureToggleService.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.hmcts.befta.featuretoggle;
+
+import io.cucumber.java.Scenario;
+import uk.gov.hmcts.befta.exception.FeatureToggleCheckFailureException;
+import uk.gov.hmcts.befta.featuretoggle.launchdarkly.LaunchDarklyFeatureToggleService;
+
+public class DefaultMultiSourceFeatureToggleService implements FeatureToggleService<Scenario, ScenarioFeatureToggleInfo> {
+
+    public static final DefaultMultiSourceFeatureToggleService INSTANCE = new DefaultMultiSourceFeatureToggleService();
+    private static final String LAUNCH_DARKLY_FLAG = "FeatureToggle";
```

Comment:
This is now a more generic one. Please rename to just `FEATURE_TOGGLE_SIGNER`.

Reply:
Updated the variable name.